packages/theme/src/theme/default.ts

```diff
@@ -52,6 +52,7 @@ export const theme = {
     colorIconWeak: "#64748B",
     colorIconStrong: "#334155",
     colorIconStronger: "#0F172A",
+    colorAvatarInitials: "#4A63FC",
```

Comment:
@richbachman the idea is to add the colors here or just keep the ones we already have? Maybe this one we can use this directly on the component because I don't believe that it will be used in another place.

Reply:
We'll be adding to these as components get built out. I'm cool with having a component specific token color here. Since its most likely the only place it will be used. 👍